bpf/lib/dbg.h

```diff
@@ -123,6 +123,7 @@ enum {
 				 * arg2: daddr (last 4 bytes for IPv6)
 				 * arg3: proxy port (in host byte order)
 				 */
+	DBG_SKIP_POLICY,
```

Comment:
Could you explain why this is needed? I don't see it being used in the code anywhere

Reply:
I understood the sentence `introducing a new DBG_SKIP_POLICY enum value` as adding a new enum value.

Reply:
Ah, yes, thanks for the context. The way I understand it, there is more than just the change here needed. One also has to change the datapath to make use of the feature as outlined by Timo (calling it at the end of ipv4_to_endpoint_is_hairpin_flow()").

So I think we should either
a) Also actually use the `DBG_SKIP_POLICY` in the way suggested by Timo (i.e. extend the second commit extend the datapath), or
b) Just do the `DBG_L7_LB` change in this PR (i.e. drop the second commit)

Either are fine with me. If we go for option b), we should remove the "Fixes" reference, since issue #27826 is not complete without implementing a) too.

Reply:
@gandro thanks feedback, I think that b is best choice, so i will drop second commit

Reply:
@gandro #27798 is merged so I think A is a good choice.